### ISSUE TITLE
highlighting in "proper" aria-owns situations

### ISSUE THREAD

**Issue by pkra** [krautzource/aria-tree-walker#21]
The examples in the doc highlight using `.is-activedescendant`. Hower, in a "proper" aria-owns situaton (i.e., when one of the aria-owns IDs is not a descendant), then rules based on that selector cannot touch the non-descending aria-owns "child". 

We could add a class to all aria-own children to allow better styling.

**Comment by pkra** [comment]
A specific use case are labeled equations with MathJax where labels are kept separately from the other table content. Thus when navigating down to a row, the highlighting won't touch the label; even worse, when navigating further down we of course first end up on the label, which is visually confusing.

**Comment by pkra** [comment]
> We could add a class to all aria-own children to allow better styling.

It would have to be all owned descendants.
